DESCRIPTION

```diff
@@ -30,21 +30,36 @@ Authors@R:
         email = "PACTA4banks@rmi.org"
       )
     )
-Description: 'PACTA' for Banks is a tool that allows banks to calculate the
-  climate alignment of their corporate lending portfolios. This package is
-  designed to make it easy to install and load multiple 'PACTA' for Banks
-  packages in a single step. It also provides thorough documentation - the
-  'PACTA' for Banks cookbook - on how to run a 'PACTA' for Banks analysis. This
-  covers prerequisites for the analysis, the separate steps of running the
-  analysis, the interpretation of 'PACTA' for Banks results, and advanced use
-  cases.
+Description: 'PACTA' (Paris Agreement Capital Transition Assessment) for Banks
+  is a tool that allows banks to calculate the climate alignment of their
+  corporate lending portfolios. This package is designed to make it easy to
+  install and load multiple 'PACTA' for Banks packages in a single step. It also
+  provides thorough documentation - the 'PACTA' for Banks cookbook - on how to
```

Comment:
But the cookbook is stored in articles, not standard vignettes. That means installing the package will not "install" the cookbook. So is the cookbook really part of the package or only part of the repo?

Reply:
As I understand it, these are no longer installed with the package.

Reply:
though, one could argue that it does still provide thorough documentation *on the pkgdown website*?

Reply:
added "in an online tutorial"